pkg/controller/logcollector/logcollector_controller.go

```diff
@@ -100,6 +100,10 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		}
 	}
 
+	err = c.Watch(&source.Kind{Type: &corev1.Node{}}, &handler.EnqueueRequestForObject{})
```

Comment:
@lmm I was just noticing some reconciles happening because of (I assume) node updates so I got looking at why we have the watch (leading me here) and I think the watch could be limited to Node Creation and Deletion. It isn't like a node's OS type can change. WDYT?